code/modules/projectiles/bullet.dm

```diff
@@ -61,7 +61,7 @@ toxic - poisons
 /datum/projectile/bullet/bullet_22
 	name = "bullet"
 	power = 22
-	shot_sound = 'sound/weapons/9x19NATO.ogg' //quieter when fired from a silenced weapon!
+	shot_sound = 'sound/weapons/smallcaliber.ogg' //quieter when fired from a silenced weapon!
```

Comment:
still gotta
```suggestion
	shot_sound = "sound/weapons/smallcaliber.ogg" //quieter when fired from a silenced weapon!
```

Reply:
I'm cool with committing that suggestion but I just want to point out that every other instance of `shot_sound` is set up with `'sound.ogg'` rather thant `"sound.ogg"`

If that's the incorrect way of doing things for the sound cache, then we haven't been utilizing it likely for years.